src/server.ts

```diff
@@ -18,6 +20,22 @@ app.use('/posts', postRouter);
 app.use('/comments', commentRouter);
 app.use('/users', userRouter);
 
+const options = {
```

Comment:
I would move it to a swagger config file

Reply:
done